src/renderer/components/helpers/ChatMethods.ts

```diff
@@ -94,10 +95,9 @@ export function openMapDialog(
 
 export async function openViewProfileDialog(
   screenContext: unwrapContext<typeof ScreenContext>,
-  contact: number | JsonContact
+  contact_id: number
```

Comment:
Are you sure that we never call it with just a contact?

Reply:
yes I changed it everywhere